src/Details/Details.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import '../scss/_Details.scss'
+
+const Details = ({ releaseDate, overview, genres, budget, revenue, runtime }) => {
+	return (
+		<section className='MovieDetails-Details'>
+			<p>Synopsis: {overview}</p>
+			<p>Runtime: {runtime}</p>
+			<p>Genres: {genres}</p>
+			<p>Release Date: {releaseDate}</p>
+			<p>Budget: {budget}</p>
+			<p>Revenue: {revenue}</p>
+		</section>
+	)
+}
```

Comment:
Honestly I can't really think of a reason not to just use p tags here, as these are all really text blocks of the same importance.